src/ipc/StrandCoord.h

```diff
@@ -32,16 +33,20 @@ class StrandCoord
     String tag; ///< optional unique well-known key (e.g., cache_dir path)
 };
 
-/// strand registration with Coordinator (also used as an ACK)
-class HereIamMessage
+/// an IPC message carrying just the kid coordinates and the message kind
+class StrandMessage
 {
 public:
-    explicit HereIamMessage(const StrandCoord &strand); ///< from registrant
-    explicit HereIamMessage(const TypedMsgHdr &hdrMsg); ///< from recvmsg()
-    void pack(TypedMsgHdr &hdrMsg) const; ///< prepare for sendmsg()
+    StrandMessage(const Ipc::MessageType, const StrandCoord &);
```

Comment:
I believe Squid tends to use `const` for integral/POD types only when defining the function:

```suggestion
    StrandMessage(Ipc::MessageType, const StrandCoord &);
```

Some implementations may want to modify their actual parameters. The caller, of course, cannot care whether the implementation modifies the _copy_ of the caller-passed POD. Thus, it kind of make sense to reserve that `const` "noise" for the function implementation, hiding it from the caller.